static/app/components/tagsTableRow.tsx

```diff
@@ -50,6 +55,18 @@ function TagsTableRow({tag, query, generateUrl, meta}: Props) {
           <StyledTooltip title={t('This tag is in the current filter conditions')}>
             <ValueContainer>{renderTagValue()}</ValueContainer>
           </StyledTooltip>
+        ) : tag.key === 'url' ? (
```

Comment:
Can add `case` here later if there are more special cases, or move it to the render function.